[2025-10-16T17:24:00.609+0000] {processor.py:157} INFO - Started process (PID=75) to work on /opt/airflow/dags/chromedriver.py
[2025-10-16T17:24:00.614+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/chromedriver.py for tasks to queue
[2025-10-16T17:24:00.631+0000] {logging_mixin.py:151} INFO - [2025-10-16T17:24:00.630+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/chromedriver.py
[2025-10-16T17:24:00.832+0000] {logging_mixin.py:151} INFO - [2025-10-16T17:24:00.792+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/chromedriver.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/chromedriver.py", line 6, in <module>
    from py_files.get import ChromebrowserOption, OpenChromeBrowser, StockResults
  File "/opt/airflow/dags/py_files/get.py", line 5, in <module>
    import selenium
ModuleNotFoundError: No module named 'selenium'
[2025-10-16T17:24:00.854+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/chromedriver.py
[2025-10-16T17:24:01.230+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/chromedriver.py took 0.637 seconds
[2025-10-16T17:24:52.404+0000] {processor.py:157} INFO - Started process (PID=142) to work on /opt/airflow/dags/chromedriver.py
[2025-10-16T17:24:52.408+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/chromedriver.py for tasks to queue
[2025-10-16T17:24:52.424+0000] {logging_mixin.py:151} INFO - [2025-10-16T17:24:52.424+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/chromedriver.py
[2025-10-16T17:24:52.735+0000] {logging_mixin.py:151} INFO - [2025-10-16T17:24:52.613+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/chromedriver.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/chromedriver.py", line 6, in <module>
    from py_files.get import ChromebrowserOption, OpenChromeBrowser, StockResults
  File "/opt/airflow/dags/py_files/get.py", line 5, in <module>
    import selenium
ModuleNotFoundError: No module named 'selenium'
[2025-10-16T17:24:52.765+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/chromedriver.py
[2025-10-16T17:24:52.870+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/chromedriver.py took 0.487 seconds
[2025-10-16T17:25:26.014+0000] {processor.py:157} INFO - Started process (PID=201) to work on /opt/airflow/dags/chromedriver.py
[2025-10-16T17:25:26.029+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/chromedriver.py for tasks to queue
[2025-10-16T17:25:26.036+0000] {logging_mixin.py:151} INFO - [2025-10-16T17:25:26.035+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/chromedriver.py
[2025-10-16T17:25:26.428+0000] {logging_mixin.py:151} INFO - [2025-10-16T17:25:26.372+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/chromedriver.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/chromedriver.py", line 6, in <module>
    from py_files.get import ChromebrowserOption, OpenChromeBrowser, StockResults
  File "/opt/airflow/dags/py_files/get.py", line 5, in <module>
    import selenium
ModuleNotFoundError: No module named 'selenium'
[2025-10-16T17:25:26.460+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/chromedriver.py
[2025-10-16T17:25:26.560+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/chromedriver.py took 0.568 seconds
[2025-10-16T17:25:57.036+0000] {processor.py:157} INFO - Started process (PID=258) to work on /opt/airflow/dags/chromedriver.py
[2025-10-16T17:25:57.039+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/chromedriver.py for tasks to queue
[2025-10-16T17:25:57.047+0000] {logging_mixin.py:151} INFO - [2025-10-16T17:25:57.046+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/chromedriver.py
[2025-10-16T17:25:57.197+0000] {logging_mixin.py:151} INFO - [2025-10-16T17:25:57.160+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/chromedriver.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/chromedriver.py", line 6, in <module>
    from py_files.get import ChromebrowserOption, OpenChromeBrowser, StockResults
  File "/opt/airflow/dags/py_files/get.py", line 5, in <module>
    import selenium
ModuleNotFoundError: No module named 'selenium'
[2025-10-16T17:25:57.212+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/chromedriver.py
[2025-10-16T17:25:57.387+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/chromedriver.py took 0.362 seconds
[2025-10-16T17:26:27.678+0000] {processor.py:157} INFO - Started process (PID=315) to work on /opt/airflow/dags/chromedriver.py
[2025-10-16T17:26:27.684+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/chromedriver.py for tasks to queue
[2025-10-16T17:26:27.700+0000] {logging_mixin.py:151} INFO - [2025-10-16T17:26:27.698+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/chromedriver.py
[2025-10-16T17:26:27.898+0000] {logging_mixin.py:151} INFO - [2025-10-16T17:26:27.846+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/chromedriver.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/chromedriver.py", line 6, in <module>
    from py_files.get import ChromebrowserOption, OpenChromeBrowser, StockResults
  File "/opt/airflow/dags/py_files/get.py", line 5, in <module>
    import selenium
ModuleNotFoundError: No module named 'selenium'
[2025-10-16T17:26:27.913+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/chromedriver.py
[2025-10-16T17:26:28.056+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/chromedriver.py took 0.396 seconds
[2025-10-16T17:26:58.440+0000] {processor.py:157} INFO - Started process (PID=374) to work on /opt/airflow/dags/chromedriver.py
[2025-10-16T17:26:58.443+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/chromedriver.py for tasks to queue
[2025-10-16T17:26:58.451+0000] {logging_mixin.py:151} INFO - [2025-10-16T17:26:58.450+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/chromedriver.py
[2025-10-16T17:26:58.538+0000] {logging_mixin.py:151} INFO - [2025-10-16T17:26:58.518+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/chromedriver.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/chromedriver.py", line 6, in <module>
    from py_files.get import ChromebrowserOption, OpenChromeBrowser, StockResults
  File "/opt/airflow/dags/py_files/get.py", line 5, in <module>
    import selenium
ModuleNotFoundError: No module named 'selenium'
[2025-10-16T17:26:58.545+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/chromedriver.py
[2025-10-16T17:26:58.604+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/chromedriver.py took 0.173 seconds
[2025-10-16T17:27:29.118+0000] {processor.py:157} INFO - Started process (PID=433) to work on /opt/airflow/dags/chromedriver.py
[2025-10-16T17:27:29.122+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/chromedriver.py for tasks to queue
[2025-10-16T17:27:29.130+0000] {logging_mixin.py:151} INFO - [2025-10-16T17:27:29.129+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/chromedriver.py
[2025-10-16T17:27:29.242+0000] {logging_mixin.py:151} INFO - [2025-10-16T17:27:29.215+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/chromedriver.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/chromedriver.py", line 6, in <module>
    from py_files.get import ChromebrowserOption, OpenChromeBrowser, StockResults
  File "/opt/airflow/dags/py_files/get.py", line 5, in <module>
    import selenium
ModuleNotFoundError: No module named 'selenium'
[2025-10-16T17:27:29.252+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/chromedriver.py
[2025-10-16T17:27:29.319+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/chromedriver.py took 0.212 seconds
[2025-10-16T17:28:00.164+0000] {processor.py:157} INFO - Started process (PID=492) to work on /opt/airflow/dags/chromedriver.py
[2025-10-16T17:28:00.168+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/chromedriver.py for tasks to queue
[2025-10-16T17:28:00.177+0000] {logging_mixin.py:151} INFO - [2025-10-16T17:28:00.177+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/chromedriver.py
[2025-10-16T17:28:00.256+0000] {logging_mixin.py:151} INFO - [2025-10-16T17:28:00.236+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/chromedriver.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/chromedriver.py", line 6, in <module>
    from py_files.get import ChromebrowserOption, OpenChromeBrowser, StockResults
  File "/opt/airflow/dags/py_files/get.py", line 5, in <module>
    import selenium
ModuleNotFoundError: No module named 'selenium'
[2025-10-16T17:28:00.265+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/chromedriver.py
[2025-10-16T17:28:00.335+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/chromedriver.py took 0.187 seconds
[2025-10-16T17:28:30.645+0000] {processor.py:157} INFO - Started process (PID=551) to work on /opt/airflow/dags/chromedriver.py
[2025-10-16T17:28:30.648+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/chromedriver.py for tasks to queue
[2025-10-16T17:28:30.655+0000] {logging_mixin.py:151} INFO - [2025-10-16T17:28:30.654+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/chromedriver.py
[2025-10-16T17:28:30.747+0000] {logging_mixin.py:151} INFO - [2025-10-16T17:28:30.727+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/chromedriver.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/chromedriver.py", line 6, in <module>
    from py_files.get import ChromebrowserOption, OpenChromeBrowser, StockResults
  File "/opt/airflow/dags/py_files/get.py", line 5, in <module>
    import selenium
ModuleNotFoundError: No module named 'selenium'
[2025-10-16T17:28:30.758+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/chromedriver.py
[2025-10-16T17:28:30.809+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/chromedriver.py took 0.175 seconds
[2025-10-16T17:29:01.308+0000] {processor.py:157} INFO - Started process (PID=610) to work on /opt/airflow/dags/chromedriver.py
[2025-10-16T17:29:01.342+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/chromedriver.py for tasks to queue
[2025-10-16T17:29:01.391+0000] {logging_mixin.py:151} INFO - [2025-10-16T17:29:01.390+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/chromedriver.py
[2025-10-16T17:29:01.582+0000] {logging_mixin.py:151} INFO - [2025-10-16T17:29:01.514+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/chromedriver.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/chromedriver.py", line 6, in <module>
    from py_files.get import ChromebrowserOption, OpenChromeBrowser, StockResults
  File "/opt/airflow/dags/py_files/get.py", line 5, in <module>
    import selenium
ModuleNotFoundError: No module named 'selenium'
[2025-10-16T17:29:01.597+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/chromedriver.py
[2025-10-16T17:29:02.631+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/chromedriver.py took 1.412 seconds
[2025-10-16T17:29:33.184+0000] {processor.py:157} INFO - Started process (PID=669) to work on /opt/airflow/dags/chromedriver.py
[2025-10-16T17:29:33.187+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/chromedriver.py for tasks to queue
[2025-10-16T17:29:33.193+0000] {logging_mixin.py:151} INFO - [2025-10-16T17:29:33.192+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/chromedriver.py
[2025-10-16T17:29:33.269+0000] {logging_mixin.py:151} INFO - [2025-10-16T17:29:33.253+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/chromedriver.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/chromedriver.py", line 6, in <module>
    from py_files.get import ChromebrowserOption, OpenChromeBrowser, StockResults
  File "/opt/airflow/dags/py_files/get.py", line 5, in <module>
    import selenium
ModuleNotFoundError: No module named 'selenium'
[2025-10-16T17:29:33.274+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/chromedriver.py
[2025-10-16T17:29:33.316+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/chromedriver.py took 0.142 seconds
[2025-10-16T17:30:03.915+0000] {processor.py:157} INFO - Started process (PID=728) to work on /opt/airflow/dags/chromedriver.py
[2025-10-16T17:30:03.923+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/chromedriver.py for tasks to queue
[2025-10-16T17:30:03.943+0000] {logging_mixin.py:151} INFO - [2025-10-16T17:30:03.942+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/chromedriver.py
[2025-10-16T17:30:04.043+0000] {logging_mixin.py:151} INFO - [2025-10-16T17:30:04.024+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/chromedriver.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/chromedriver.py", line 6, in <module>
    from py_files.get import ChromebrowserOption, OpenChromeBrowser, StockResults
  File "/opt/airflow/dags/py_files/get.py", line 5, in <module>
    import selenium
ModuleNotFoundError: No module named 'selenium'
[2025-10-16T17:30:04.048+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/chromedriver.py
[2025-10-16T17:30:04.341+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/chromedriver.py took 0.493 seconds
[2025-10-16T17:30:35.333+0000] {processor.py:157} INFO - Started process (PID=787) to work on /opt/airflow/dags/chromedriver.py
[2025-10-16T17:30:35.335+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/chromedriver.py for tasks to queue
[2025-10-16T17:30:35.343+0000] {logging_mixin.py:151} INFO - [2025-10-16T17:30:35.342+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/chromedriver.py
[2025-10-16T17:30:35.415+0000] {logging_mixin.py:151} INFO - [2025-10-16T17:30:35.400+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/chromedriver.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/chromedriver.py", line 6, in <module>
    from py_files.get import ChromebrowserOption, OpenChromeBrowser, StockResults
  File "/opt/airflow/dags/py_files/get.py", line 5, in <module>
    import selenium
ModuleNotFoundError: No module named 'selenium'
[2025-10-16T17:30:35.421+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/chromedriver.py
[2025-10-16T17:30:35.466+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/chromedriver.py took 0.144 seconds
[2025-10-16T17:31:05.646+0000] {processor.py:157} INFO - Started process (PID=846) to work on /opt/airflow/dags/chromedriver.py
[2025-10-16T17:31:05.649+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/chromedriver.py for tasks to queue
[2025-10-16T17:31:05.657+0000] {logging_mixin.py:151} INFO - [2025-10-16T17:31:05.656+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/chromedriver.py
[2025-10-16T17:31:05.793+0000] {logging_mixin.py:151} INFO - [2025-10-16T17:31:05.746+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/chromedriver.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/chromedriver.py", line 6, in <module>
    from py_files.get import ChromebrowserOption, OpenChromeBrowser, StockResults
  File "/opt/airflow/dags/py_files/get.py", line 5, in <module>
    import selenium
ModuleNotFoundError: No module named 'selenium'
[2025-10-16T17:31:05.813+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/chromedriver.py
[2025-10-16T17:31:05.956+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/chromedriver.py took 0.319 seconds
[2025-10-16T17:31:36.093+0000] {processor.py:157} INFO - Started process (PID=906) to work on /opt/airflow/dags/chromedriver.py
[2025-10-16T17:31:36.095+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/chromedriver.py for tasks to queue
[2025-10-16T17:31:36.101+0000] {logging_mixin.py:151} INFO - [2025-10-16T17:31:36.101+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/chromedriver.py
[2025-10-16T17:31:36.194+0000] {logging_mixin.py:151} INFO - [2025-10-16T17:31:36.175+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/chromedriver.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/chromedriver.py", line 6, in <module>
    from py_files.get import ChromebrowserOption, OpenChromeBrowser, StockResults
  File "/opt/airflow/dags/py_files/get.py", line 5, in <module>
    import selenium
ModuleNotFoundError: No module named 'selenium'
[2025-10-16T17:31:36.200+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/chromedriver.py
[2025-10-16T17:31:36.242+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/chromedriver.py took 0.157 seconds
[2025-10-16T17:32:06.999+0000] {processor.py:157} INFO - Started process (PID=965) to work on /opt/airflow/dags/chromedriver.py
[2025-10-16T17:32:07.004+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/chromedriver.py for tasks to queue
[2025-10-16T17:32:07.019+0000] {logging_mixin.py:151} INFO - [2025-10-16T17:32:07.017+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/chromedriver.py
[2025-10-16T17:32:07.184+0000] {logging_mixin.py:151} INFO - [2025-10-16T17:32:07.154+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/chromedriver.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/chromedriver.py", line 6, in <module>
    from py_files.get import ChromebrowserOption, OpenChromeBrowser, StockResults
  File "/opt/airflow/dags/py_files/get.py", line 5, in <module>
    import selenium
ModuleNotFoundError: No module named 'selenium'
[2025-10-16T17:32:07.199+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/chromedriver.py
[2025-10-16T17:32:07.269+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/chromedriver.py took 0.290 seconds
[2025-10-16T17:32:37.702+0000] {processor.py:157} INFO - Started process (PID=1024) to work on /opt/airflow/dags/chromedriver.py
[2025-10-16T17:32:37.705+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/chromedriver.py for tasks to queue
[2025-10-16T17:32:37.712+0000] {logging_mixin.py:151} INFO - [2025-10-16T17:32:37.711+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/chromedriver.py
[2025-10-16T17:32:38.111+0000] {logging_mixin.py:151} INFO - [2025-10-16T17:32:38.096+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/chromedriver.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/chromedriver.py", line 6, in <module>
    from py_files.get import ChromebrowserOption, OpenChromeBrowser, StockResults
  File "/opt/airflow/dags/py_files/get.py", line 3
    from home.airflow..local.lib.python3.8.site-packages import pandas
                      ^
SyntaxError: invalid syntax
[2025-10-16T17:32:38.116+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/chromedriver.py
[2025-10-16T17:32:38.200+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/chromedriver.py took 0.511 seconds
[2025-10-16T17:33:08.795+0000] {processor.py:157} INFO - Started process (PID=1083) to work on /opt/airflow/dags/chromedriver.py
[2025-10-16T17:33:08.798+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/chromedriver.py for tasks to queue
[2025-10-16T17:33:08.814+0000] {logging_mixin.py:151} INFO - [2025-10-16T17:33:08.813+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/chromedriver.py
[2025-10-16T17:33:08.937+0000] {logging_mixin.py:151} INFO - [2025-10-16T17:33:08.927+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/chromedriver.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/chromedriver.py", line 6, in <module>
    from py_files.get import ChromebrowserOption, OpenChromeBrowser, StockResults
  File "/opt/airflow/dags/py_files/get.py", line 3
    from home.airflow..local.lib.python3.8.site-packages import pandas
                      ^
SyntaxError: invalid syntax
[2025-10-16T17:33:08.941+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/chromedriver.py
[2025-10-16T17:33:08.993+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/chromedriver.py took 0.218 seconds
[2025-10-16T17:33:39.989+0000] {processor.py:157} INFO - Started process (PID=1142) to work on /opt/airflow/dags/chromedriver.py
[2025-10-16T17:33:39.992+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/chromedriver.py for tasks to queue
[2025-10-16T17:33:40.000+0000] {logging_mixin.py:151} INFO - [2025-10-16T17:33:39.999+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/chromedriver.py
[2025-10-16T17:33:40.096+0000] {logging_mixin.py:151} INFO - [2025-10-16T17:33:40.086+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/chromedriver.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/chromedriver.py", line 6, in <module>
    from py_files.get import ChromebrowserOption, OpenChromeBrowser, StockResults
  File "/opt/airflow/dags/py_files/get.py", line 3
    from site-packages import pandas
             ^
SyntaxError: invalid syntax
[2025-10-16T17:33:40.100+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/chromedriver.py
[2025-10-16T17:33:40.158+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/chromedriver.py took 0.180 seconds
[2025-10-16T17:34:07.613+0000] {processor.py:157} INFO - Started process (PID=1193) to work on /opt/airflow/dags/chromedriver.py
[2025-10-16T17:34:07.616+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/chromedriver.py for tasks to queue
[2025-10-16T17:34:07.623+0000] {logging_mixin.py:151} INFO - [2025-10-16T17:34:07.622+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/chromedriver.py
[2025-10-16T17:34:07.790+0000] {logging_mixin.py:151} INFO - [2025-10-16T17:34:07.777+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/chromedriver.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/chromedriver.py", line 6, in <module>
    from py_files.get import ChromebrowserOption, OpenChromeBrowser, StockResults
  File "/opt/airflow/dags/py_files/get.py", line 3
    from site-packages import pandas
             ^
SyntaxError: invalid syntax
[2025-10-16T17:34:07.795+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/chromedriver.py
[2025-10-16T17:34:07.854+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/chromedriver.py took 0.255 seconds
[2025-10-16T17:34:14.657+0000] {processor.py:157} INFO - Started process (PID=1211) to work on /opt/airflow/dags/chromedriver.py
[2025-10-16T17:34:14.660+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/chromedriver.py for tasks to queue
[2025-10-16T17:34:14.668+0000] {logging_mixin.py:151} INFO - [2025-10-16T17:34:14.667+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/chromedriver.py
[2025-10-16T17:34:14.947+0000] {logging_mixin.py:151} INFO - [2025-10-16T17:34:14.917+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/chromedriver.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/chromedriver.py", line 6, in <module>
    from py_files.get import ChromebrowserOption, OpenChromeBrowser, StockResults
  File "/opt/airflow/dags/py_files/get.py", line 3
    from site-packages import pandas
             ^
SyntaxError: invalid syntax
[2025-10-16T17:34:14.956+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/chromedriver.py
[2025-10-16T17:34:15.047+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/chromedriver.py took 0.401 seconds
[2025-10-16T17:34:15.766+0000] {processor.py:157} INFO - Started process (PID=1213) to work on /opt/airflow/dags/chromedriver.py
[2025-10-16T17:34:15.776+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/chromedriver.py for tasks to queue
[2025-10-16T17:34:15.787+0000] {logging_mixin.py:151} INFO - [2025-10-16T17:34:15.785+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/chromedriver.py
[2025-10-16T17:34:15.909+0000] {logging_mixin.py:151} INFO - [2025-10-16T17:34:15.898+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/chromedriver.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/chromedriver.py", line 6, in <module>
    from py_files.get import ChromebrowserOption, OpenChromeBrowser, StockResults
  File "/opt/airflow/dags/py_files/get.py", line 3
    from site-packages import pandas
             ^
SyntaxError: invalid syntax
[2025-10-16T17:34:15.912+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/chromedriver.py
[2025-10-16T17:34:15.988+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/chromedriver.py took 0.235 seconds
[2025-10-16T17:34:46.376+0000] {processor.py:157} INFO - Started process (PID=1272) to work on /opt/airflow/dags/chromedriver.py
[2025-10-16T17:34:46.380+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/chromedriver.py for tasks to queue
[2025-10-16T17:34:46.397+0000] {logging_mixin.py:151} INFO - [2025-10-16T17:34:46.396+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/chromedriver.py
[2025-10-16T17:34:46.503+0000] {logging_mixin.py:151} INFO - [2025-10-16T17:34:46.492+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/chromedriver.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/chromedriver.py", line 6, in <module>
    from py_files.get import ChromebrowserOption, OpenChromeBrowser, StockResults
  File "/opt/airflow/dags/py_files/get.py", line 3
    from site-packages import pandas
             ^
SyntaxError: invalid syntax
[2025-10-16T17:34:46.507+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/chromedriver.py
[2025-10-16T17:34:46.561+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/chromedriver.py took 0.205 seconds
[2025-10-16T17:35:16.750+0000] {processor.py:157} INFO - Started process (PID=1331) to work on /opt/airflow/dags/chromedriver.py
[2025-10-16T17:35:16.753+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/chromedriver.py for tasks to queue
[2025-10-16T17:35:16.759+0000] {logging_mixin.py:151} INFO - [2025-10-16T17:35:16.759+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/chromedriver.py
[2025-10-16T17:35:16.855+0000] {logging_mixin.py:151} INFO - [2025-10-16T17:35:16.840+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/chromedriver.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/chromedriver.py", line 6, in <module>
    from py_files.get import ChromebrowserOption, OpenChromeBrowser, StockResults
  File "/opt/airflow/dags/py_files/get.py", line 3
    from site-packages import pandas
             ^
SyntaxError: invalid syntax
[2025-10-16T17:35:16.863+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/chromedriver.py
[2025-10-16T17:35:16.915+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/chromedriver.py took 0.173 seconds
[2025-10-16T17:35:47.634+0000] {processor.py:157} INFO - Started process (PID=1389) to work on /opt/airflow/dags/chromedriver.py
[2025-10-16T17:35:47.636+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/chromedriver.py for tasks to queue
[2025-10-16T17:35:47.642+0000] {logging_mixin.py:151} INFO - [2025-10-16T17:35:47.641+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/chromedriver.py
[2025-10-16T17:35:47.731+0000] {logging_mixin.py:151} INFO - [2025-10-16T17:35:47.721+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/chromedriver.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/chromedriver.py", line 6, in <module>
    from py_files.get import ChromebrowserOption, OpenChromeBrowser, StockResults
  File "/opt/airflow/dags/py_files/get.py", line 3
    from site-packages import pandas
             ^
SyntaxError: invalid syntax
[2025-10-16T17:35:47.734+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/chromedriver.py
[2025-10-16T17:35:47.779+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/chromedriver.py took 0.154 seconds
[2025-10-16T17:36:18.128+0000] {processor.py:157} INFO - Started process (PID=1448) to work on /opt/airflow/dags/chromedriver.py
[2025-10-16T17:36:18.137+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/chromedriver.py for tasks to queue
[2025-10-16T17:36:18.144+0000] {logging_mixin.py:151} INFO - [2025-10-16T17:36:18.143+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/chromedriver.py
[2025-10-16T17:36:18.243+0000] {logging_mixin.py:151} INFO - [2025-10-16T17:36:18.230+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/chromedriver.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/chromedriver.py", line 6, in <module>
    from py_files.get import ChromebrowserOption, OpenChromeBrowser, StockResults
  File "/opt/airflow/dags/py_files/get.py", line 3
    from site-packages import pandas
             ^
SyntaxError: invalid syntax
[2025-10-16T17:36:18.247+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/chromedriver.py
[2025-10-16T17:36:18.294+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/chromedriver.py took 0.176 seconds
[2025-10-16T17:36:49.180+0000] {processor.py:157} INFO - Started process (PID=1507) to work on /opt/airflow/dags/chromedriver.py
[2025-10-16T17:36:49.183+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/chromedriver.py for tasks to queue
[2025-10-16T17:36:49.190+0000] {logging_mixin.py:151} INFO - [2025-10-16T17:36:49.189+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/chromedriver.py
[2025-10-16T17:36:49.320+0000] {logging_mixin.py:151} INFO - [2025-10-16T17:36:49.308+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/chromedriver.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/chromedriver.py", line 6, in <module>
    from py_files.get import ChromebrowserOption, OpenChromeBrowser, StockResults
  File "/opt/airflow/dags/py_files/get.py", line 3
    from site-packages import pandas
             ^
SyntaxError: invalid syntax
[2025-10-16T17:36:49.325+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/chromedriver.py
[2025-10-16T17:36:49.377+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/chromedriver.py took 0.208 seconds
[2025-10-16T17:37:20.326+0000] {processor.py:157} INFO - Started process (PID=1567) to work on /opt/airflow/dags/chromedriver.py
[2025-10-16T17:37:20.331+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/chromedriver.py for tasks to queue
[2025-10-16T17:37:20.345+0000] {logging_mixin.py:151} INFO - [2025-10-16T17:37:20.343+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/chromedriver.py
[2025-10-16T17:37:20.562+0000] {logging_mixin.py:151} INFO - [2025-10-16T17:37:20.538+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/chromedriver.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/chromedriver.py", line 6, in <module>
    from py_files.get import ChromebrowserOption, OpenChromeBrowser, StockResults
  File "/opt/airflow/dags/py_files/get.py", line 3
    from site-packages import pandas
             ^
SyntaxError: invalid syntax
[2025-10-16T17:37:20.611+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/chromedriver.py
[2025-10-16T17:37:20.675+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/chromedriver.py took 0.368 seconds
[2025-10-16T17:37:51.096+0000] {processor.py:157} INFO - Started process (PID=1626) to work on /opt/airflow/dags/chromedriver.py
[2025-10-16T17:37:51.101+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/chromedriver.py for tasks to queue
[2025-10-16T17:37:51.111+0000] {logging_mixin.py:151} INFO - [2025-10-16T17:37:51.109+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/chromedriver.py
[2025-10-16T17:37:51.255+0000] {logging_mixin.py:151} INFO - [2025-10-16T17:37:51.235+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/chromedriver.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/chromedriver.py", line 6, in <module>
    from py_files.get import ChromebrowserOption, OpenChromeBrowser, StockResults
  File "/opt/airflow/dags/py_files/get.py", line 3
    from site-packages import pandas
             ^
SyntaxError: invalid syntax
[2025-10-16T17:37:51.259+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/chromedriver.py
[2025-10-16T17:37:51.636+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/chromedriver.py took 0.561 seconds
[2025-10-16T17:38:21.936+0000] {processor.py:157} INFO - Started process (PID=1683) to work on /opt/airflow/dags/chromedriver.py
[2025-10-16T17:38:21.943+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/chromedriver.py for tasks to queue
[2025-10-16T17:38:21.965+0000] {logging_mixin.py:151} INFO - [2025-10-16T17:38:21.963+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/chromedriver.py
[2025-10-16T17:38:22.233+0000] {logging_mixin.py:151} INFO - [2025-10-16T17:38:22.205+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/chromedriver.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/chromedriver.py", line 6, in <module>
    from py_files.get import ChromebrowserOption, OpenChromeBrowser, StockResults
  File "/opt/airflow/dags/py_files/get.py", line 3
    from site-packages import pandas
             ^
SyntaxError: invalid syntax
[2025-10-16T17:38:22.242+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/chromedriver.py
[2025-10-16T17:38:22.322+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/chromedriver.py took 0.406 seconds
[2025-10-16T17:38:52.716+0000] {processor.py:157} INFO - Started process (PID=1742) to work on /opt/airflow/dags/chromedriver.py
[2025-10-16T17:38:52.722+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/chromedriver.py for tasks to queue
[2025-10-16T17:38:52.737+0000] {logging_mixin.py:151} INFO - [2025-10-16T17:38:52.735+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/chromedriver.py
[2025-10-16T17:38:52.951+0000] {logging_mixin.py:151} INFO - [2025-10-16T17:38:52.927+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/chromedriver.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/chromedriver.py", line 6, in <module>
    from py_files.get import ChromebrowserOption, OpenChromeBrowser, StockResults
  File "/opt/airflow/dags/py_files/get.py", line 3
    from site-packages import pandas
             ^
SyntaxError: invalid syntax
[2025-10-16T17:38:52.958+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/chromedriver.py
[2025-10-16T17:38:53.072+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/chromedriver.py took 0.377 seconds
[2025-10-16T17:39:23.734+0000] {processor.py:157} INFO - Started process (PID=1801) to work on /opt/airflow/dags/chromedriver.py
[2025-10-16T17:39:23.739+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/chromedriver.py for tasks to queue
[2025-10-16T17:39:23.748+0000] {logging_mixin.py:151} INFO - [2025-10-16T17:39:23.747+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/chromedriver.py
[2025-10-16T17:39:23.878+0000] {logging_mixin.py:151} INFO - [2025-10-16T17:39:23.866+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/chromedriver.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/chromedriver.py", line 6, in <module>
    from py_files.get import ChromebrowserOption, OpenChromeBrowser, StockResults
  File "/opt/airflow/dags/py_files/get.py", line 3
    from site-packages import pandas
             ^
SyntaxError: invalid syntax
[2025-10-16T17:39:23.883+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/chromedriver.py
[2025-10-16T17:39:23.938+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/chromedriver.py took 0.216 seconds
[2025-10-16T17:39:54.500+0000] {processor.py:157} INFO - Started process (PID=1860) to work on /opt/airflow/dags/chromedriver.py
[2025-10-16T17:39:54.504+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/chromedriver.py for tasks to queue
[2025-10-16T17:39:54.513+0000] {logging_mixin.py:151} INFO - [2025-10-16T17:39:54.512+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/chromedriver.py
[2025-10-16T17:39:54.767+0000] {logging_mixin.py:151} INFO - [2025-10-16T17:39:54.741+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/chromedriver.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/chromedriver.py", line 6, in <module>
    from py_files.get import ChromebrowserOption, OpenChromeBrowser, StockResults
  File "/opt/airflow/dags/py_files/get.py", line 3, in <module>
    import webdriver
ModuleNotFoundError: No module named 'webdriver'
[2025-10-16T17:39:54.774+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/chromedriver.py
[2025-10-16T17:39:54.871+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/chromedriver.py took 0.397 seconds
[2025-10-16T17:40:25.275+0000] {processor.py:157} INFO - Started process (PID=1917) to work on /opt/airflow/dags/chromedriver.py
[2025-10-16T17:40:25.279+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/chromedriver.py for tasks to queue
[2025-10-16T17:40:25.286+0000] {logging_mixin.py:151} INFO - [2025-10-16T17:40:25.285+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/chromedriver.py
[2025-10-16T17:40:25.426+0000] {logging_mixin.py:151} INFO - [2025-10-16T17:40:25.392+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/chromedriver.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/chromedriver.py", line 6, in <module>
    from py_files.get import ChromebrowserOption, OpenChromeBrowser, StockResults
  File "/opt/airflow/dags/py_files/get.py", line 3, in <module>
    import webdriver
ModuleNotFoundError: No module named 'webdriver'
[2025-10-16T17:40:25.436+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/chromedriver.py
[2025-10-16T17:40:25.488+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/chromedriver.py took 0.223 seconds
[2025-10-16T17:40:55.807+0000] {processor.py:157} INFO - Started process (PID=1976) to work on /opt/airflow/dags/chromedriver.py
[2025-10-16T17:40:55.812+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/chromedriver.py for tasks to queue
[2025-10-16T17:40:55.825+0000] {logging_mixin.py:151} INFO - [2025-10-16T17:40:55.824+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/chromedriver.py
[2025-10-16T17:40:56.068+0000] {logging_mixin.py:151} INFO - [2025-10-16T17:40:56.022+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/chromedriver.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/chromedriver.py", line 6, in <module>
    from py_files.get import ChromebrowserOption, OpenChromeBrowser, StockResults
  File "/opt/airflow/dags/py_files/get.py", line 3, in <module>
    import webdriver
ModuleNotFoundError: No module named 'webdriver'
[2025-10-16T17:40:56.083+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/chromedriver.py
[2025-10-16T17:40:56.197+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/chromedriver.py took 0.409 seconds
[2025-10-16T17:41:26.451+0000] {processor.py:157} INFO - Started process (PID=2035) to work on /opt/airflow/dags/chromedriver.py
[2025-10-16T17:41:26.454+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/chromedriver.py for tasks to queue
[2025-10-16T17:41:26.461+0000] {logging_mixin.py:151} INFO - [2025-10-16T17:41:26.460+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/chromedriver.py
[2025-10-16T17:41:26.609+0000] {logging_mixin.py:151} INFO - [2025-10-16T17:41:26.579+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/chromedriver.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/chromedriver.py", line 6, in <module>
    from py_files.get import ChromebrowserOption, OpenChromeBrowser, StockResults
  File "/opt/airflow/dags/py_files/get.py", line 3, in <module>
    import webdriver
ModuleNotFoundError: No module named 'webdriver'
[2025-10-16T17:41:26.618+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/chromedriver.py
[2025-10-16T17:41:26.671+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/chromedriver.py took 0.230 seconds
[2025-10-16T17:41:57.135+0000] {processor.py:157} INFO - Started process (PID=2094) to work on /opt/airflow/dags/chromedriver.py
[2025-10-16T17:41:57.138+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/chromedriver.py for tasks to queue
[2025-10-16T17:41:57.145+0000] {logging_mixin.py:151} INFO - [2025-10-16T17:41:57.144+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/chromedriver.py
[2025-10-16T17:41:57.266+0000] {logging_mixin.py:151} INFO - [2025-10-16T17:41:57.244+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/chromedriver.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/chromedriver.py", line 6, in <module>
    from py_files.get import ChromebrowserOption, OpenChromeBrowser, StockResults
  File "/opt/airflow/dags/py_files/get.py", line 3, in <module>
    import webdriver
ModuleNotFoundError: No module named 'webdriver'
[2025-10-16T17:41:57.275+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/chromedriver.py
[2025-10-16T17:41:57.335+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/chromedriver.py took 0.208 seconds
[2025-10-16T17:42:27.766+0000] {processor.py:157} INFO - Started process (PID=2152) to work on /opt/airflow/dags/chromedriver.py
[2025-10-16T17:42:27.768+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/chromedriver.py for tasks to queue
[2025-10-16T17:42:27.773+0000] {logging_mixin.py:151} INFO - [2025-10-16T17:42:27.772+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/chromedriver.py
[2025-10-16T17:42:27.873+0000] {logging_mixin.py:151} INFO - [2025-10-16T17:42:27.847+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/chromedriver.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/chromedriver.py", line 6, in <module>
    from py_files.get import ChromebrowserOption, OpenChromeBrowser, StockResults
  File "/opt/airflow/dags/py_files/get.py", line 3, in <module>
    import webdriver
ModuleNotFoundError: No module named 'webdriver'
[2025-10-16T17:42:27.881+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/chromedriver.py
[2025-10-16T17:42:27.968+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/chromedriver.py took 0.210 seconds
[2025-10-16T17:42:59.231+0000] {processor.py:157} INFO - Started process (PID=2211) to work on /opt/airflow/dags/chromedriver.py
[2025-10-16T17:42:59.235+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/chromedriver.py for tasks to queue
[2025-10-16T17:42:59.261+0000] {logging_mixin.py:151} INFO - [2025-10-16T17:42:59.260+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/chromedriver.py
[2025-10-16T17:42:59.454+0000] {logging_mixin.py:151} INFO - [2025-10-16T17:42:59.431+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/chromedriver.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/chromedriver.py", line 6, in <module>
    from py_files.get import ChromebrowserOption, OpenChromeBrowser, StockResults
  File "/opt/airflow/dags/py_files/get.py", line 3, in <module>
    import webdriver
ModuleNotFoundError: No module named 'webdriver'
[2025-10-16T17:42:59.461+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/chromedriver.py
[2025-10-16T17:42:59.531+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/chromedriver.py took 0.324 seconds
[2025-10-16T17:43:29.835+0000] {processor.py:157} INFO - Started process (PID=2271) to work on /opt/airflow/dags/chromedriver.py
[2025-10-16T17:43:29.837+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/chromedriver.py for tasks to queue
[2025-10-16T17:43:29.844+0000] {logging_mixin.py:151} INFO - [2025-10-16T17:43:29.843+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/chromedriver.py
[2025-10-16T17:43:29.973+0000] {logging_mixin.py:151} INFO - [2025-10-16T17:43:29.949+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/chromedriver.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/chromedriver.py", line 6, in <module>
    from py_files.get import ChromebrowserOption, OpenChromeBrowser, StockResults
  File "/opt/airflow/dags/py_files/get.py", line 3, in <module>
    import webdriver
ModuleNotFoundError: No module named 'webdriver'
[2025-10-16T17:43:29.981+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/chromedriver.py
[2025-10-16T17:43:30.039+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/chromedriver.py took 0.212 seconds
[2025-10-16T17:44:01.413+0000] {processor.py:157} INFO - Started process (PID=2330) to work on /opt/airflow/dags/chromedriver.py
[2025-10-16T17:44:01.419+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/chromedriver.py for tasks to queue
[2025-10-16T17:44:01.432+0000] {logging_mixin.py:151} INFO - [2025-10-16T17:44:01.431+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/chromedriver.py
[2025-10-16T17:44:01.638+0000] {logging_mixin.py:151} INFO - [2025-10-16T17:44:01.610+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/chromedriver.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/chromedriver.py", line 6, in <module>
    from py_files.get import ChromebrowserOption, OpenChromeBrowser, StockResults
  File "/opt/airflow/dags/py_files/get.py", line 3, in <module>
    import webdriver
ModuleNotFoundError: No module named 'webdriver'
[2025-10-16T17:44:01.647+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/chromedriver.py
[2025-10-16T17:44:01.696+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/chromedriver.py took 0.297 seconds
[2025-10-16T17:44:31.995+0000] {processor.py:157} INFO - Started process (PID=2390) to work on /opt/airflow/dags/chromedriver.py
[2025-10-16T17:44:31.999+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/chromedriver.py for tasks to queue
[2025-10-16T17:44:32.011+0000] {logging_mixin.py:151} INFO - [2025-10-16T17:44:32.009+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/chromedriver.py
[2025-10-16T17:44:32.156+0000] {logging_mixin.py:151} INFO - [2025-10-16T17:44:32.127+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/chromedriver.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/chromedriver.py", line 6, in <module>
    from py_files.get import ChromebrowserOption, OpenChromeBrowser, StockResults
  File "/opt/airflow/dags/py_files/get.py", line 3, in <module>
    import webdriver
ModuleNotFoundError: No module named 'webdriver'
[2025-10-16T17:44:32.166+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/chromedriver.py
[2025-10-16T17:44:32.234+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/chromedriver.py took 0.249 seconds
[2025-10-16T17:45:02.547+0000] {processor.py:157} INFO - Started process (PID=2448) to work on /opt/airflow/dags/chromedriver.py
[2025-10-16T17:45:02.552+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/chromedriver.py for tasks to queue
[2025-10-16T17:45:02.566+0000] {logging_mixin.py:151} INFO - [2025-10-16T17:45:02.564+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/chromedriver.py
[2025-10-16T17:45:02.813+0000] {logging_mixin.py:151} INFO - [2025-10-16T17:45:02.769+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/chromedriver.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/chromedriver.py", line 6, in <module>
    from py_files.get import ChromebrowserOption, OpenChromeBrowser, StockResults
  File "/opt/airflow/dags/py_files/get.py", line 3, in <module>
    import webdriver
ModuleNotFoundError: No module named 'webdriver'
[2025-10-16T17:45:02.828+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/chromedriver.py
[2025-10-16T17:45:02.923+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/chromedriver.py took 0.401 seconds
[2025-10-16T17:45:33.215+0000] {processor.py:157} INFO - Started process (PID=2507) to work on /opt/airflow/dags/chromedriver.py
[2025-10-16T17:45:33.218+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/chromedriver.py for tasks to queue
[2025-10-16T17:45:33.225+0000] {logging_mixin.py:151} INFO - [2025-10-16T17:45:33.224+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/chromedriver.py
[2025-10-16T17:45:33.377+0000] {logging_mixin.py:151} INFO - [2025-10-16T17:45:33.346+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/chromedriver.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/chromedriver.py", line 6, in <module>
    from py_files.get import ChromebrowserOption, OpenChromeBrowser, StockResults
  File "/opt/airflow/dags/py_files/get.py", line 3, in <module>
    import webdriver
ModuleNotFoundError: No module named 'webdriver'
[2025-10-16T17:45:33.388+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/chromedriver.py
[2025-10-16T17:45:33.459+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/chromedriver.py took 0.257 seconds
[2025-10-16T17:46:03.964+0000] {processor.py:157} INFO - Started process (PID=2566) to work on /opt/airflow/dags/chromedriver.py
[2025-10-16T17:46:03.970+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/chromedriver.py for tasks to queue
[2025-10-16T17:46:03.977+0000] {logging_mixin.py:151} INFO - [2025-10-16T17:46:03.976+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/chromedriver.py
[2025-10-16T17:46:04.103+0000] {logging_mixin.py:151} INFO - [2025-10-16T17:46:04.080+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/chromedriver.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/chromedriver.py", line 6, in <module>
    from py_files.get import ChromebrowserOption, OpenChromeBrowser, StockResults
  File "/opt/airflow/dags/py_files/get.py", line 3, in <module>
    import webdriver
ModuleNotFoundError: No module named 'webdriver'
[2025-10-16T17:46:04.111+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/chromedriver.py
[2025-10-16T17:46:04.168+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/chromedriver.py took 0.216 seconds
[2025-10-16T17:46:35.095+0000] {processor.py:157} INFO - Started process (PID=2625) to work on /opt/airflow/dags/chromedriver.py
[2025-10-16T17:46:35.098+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/chromedriver.py for tasks to queue
[2025-10-16T17:46:35.103+0000] {logging_mixin.py:151} INFO - [2025-10-16T17:46:35.102+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/chromedriver.py
[2025-10-16T17:46:35.208+0000] {logging_mixin.py:151} INFO - [2025-10-16T17:46:35.187+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/chromedriver.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/chromedriver.py", line 6, in <module>
    from py_files.get import ChromebrowserOption, OpenChromeBrowser, StockResults
  File "/opt/airflow/dags/py_files/get.py", line 3, in <module>
    import webdriver
ModuleNotFoundError: No module named 'webdriver'
[2025-10-16T17:46:35.215+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/chromedriver.py
[2025-10-16T17:46:35.257+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/chromedriver.py took 0.168 seconds
[2025-10-16T17:51:36.233+0000] {processor.py:157} INFO - Started process (PID=82) to work on /opt/airflow/dags/chromedriver.py
[2025-10-16T17:51:36.237+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/chromedriver.py for tasks to queue
[2025-10-16T17:51:36.260+0000] {logging_mixin.py:151} INFO - [2025-10-16T17:51:36.256+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/chromedriver.py
[2025-10-16T17:51:36.570+0000] {logging_mixin.py:151} INFO - [2025-10-16T17:51:36.543+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/chromedriver.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/chromedriver.py", line 6, in <module>
    from py_files.get import ChromebrowserOption, OpenChromeBrowser, StockResults
  File "/opt/airflow/dags/py_files/get.py", line 3, in <module>
    import webdriver
ModuleNotFoundError: No module named 'webdriver'
[2025-10-16T17:51:36.579+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/chromedriver.py
[2025-10-16T17:51:36.678+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/chromedriver.py took 0.466 seconds
[2025-10-16T17:52:06.898+0000] {processor.py:157} INFO - Started process (PID=141) to work on /opt/airflow/dags/chromedriver.py
[2025-10-16T17:52:06.907+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/chromedriver.py for tasks to queue
[2025-10-16T17:52:06.934+0000] {logging_mixin.py:151} INFO - [2025-10-16T17:52:06.933+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/chromedriver.py
[2025-10-16T17:52:08.085+0000] {logging_mixin.py:151} INFO - [2025-10-16T17:52:08.020+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/chromedriver.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/chromedriver.py", line 6, in <module>
    from py_files.get import ChromebrowserOption, OpenChromeBrowser, StockResults
  File "/opt/airflow/dags/py_files/get.py", line 3, in <module>
    import webdriver
ModuleNotFoundError: No module named 'webdriver'
[2025-10-16T17:52:08.094+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/chromedriver.py
[2025-10-16T17:52:08.216+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/chromedriver.py took 1.332 seconds
[2025-10-16T17:52:38.474+0000] {processor.py:157} INFO - Started process (PID=200) to work on /opt/airflow/dags/chromedriver.py
[2025-10-16T17:52:38.483+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/chromedriver.py for tasks to queue
[2025-10-16T17:52:38.502+0000] {logging_mixin.py:151} INFO - [2025-10-16T17:52:38.500+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/chromedriver.py
[2025-10-16T17:52:38.767+0000] {logging_mixin.py:151} INFO - [2025-10-16T17:52:38.713+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/chromedriver.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/chromedriver.py", line 6, in <module>
    from py_files.get import ChromebrowserOption, OpenChromeBrowser, StockResults
  File "/opt/airflow/dags/py_files/get.py", line 3, in <module>
    import webdriver
ModuleNotFoundError: No module named 'webdriver'
[2025-10-16T17:52:38.785+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/chromedriver.py
[2025-10-16T17:52:38.990+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/chromedriver.py took 0.549 seconds
[2025-10-16T17:53:10.159+0000] {processor.py:157} INFO - Started process (PID=259) to work on /opt/airflow/dags/chromedriver.py
[2025-10-16T17:53:10.167+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/chromedriver.py for tasks to queue
[2025-10-16T17:53:10.181+0000] {logging_mixin.py:151} INFO - [2025-10-16T17:53:10.180+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/chromedriver.py
[2025-10-16T17:53:10.280+0000] {logging_mixin.py:151} INFO - [2025-10-16T17:53:10.262+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/chromedriver.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/chromedriver.py", line 6, in <module>
    from py_files.get import ChromebrowserOption, OpenChromeBrowser, StockResults
  File "/opt/airflow/dags/py_files/get.py", line 3, in <module>
    import webdriver
ModuleNotFoundError: No module named 'webdriver'
[2025-10-16T17:53:10.286+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/chromedriver.py
[2025-10-16T17:53:10.338+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/chromedriver.py took 0.198 seconds
[2025-10-16T17:53:40.485+0000] {processor.py:157} INFO - Started process (PID=318) to work on /opt/airflow/dags/chromedriver.py
[2025-10-16T17:53:40.489+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/chromedriver.py for tasks to queue
[2025-10-16T17:53:40.500+0000] {logging_mixin.py:151} INFO - [2025-10-16T17:53:40.498+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/chromedriver.py
[2025-10-16T17:53:40.617+0000] {logging_mixin.py:151} INFO - [2025-10-16T17:53:40.600+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/chromedriver.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/chromedriver.py", line 6, in <module>
    from py_files.get import ChromebrowserOption, OpenChromeBrowser, StockResults
  File "/opt/airflow/dags/py_files/get.py", line 3, in <module>
    import webdriver
ModuleNotFoundError: No module named 'webdriver'
[2025-10-16T17:53:40.622+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/chromedriver.py
[2025-10-16T17:53:40.668+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/chromedriver.py took 0.203 seconds
[2025-10-16T17:54:10.929+0000] {processor.py:157} INFO - Started process (PID=375) to work on /opt/airflow/dags/chromedriver.py
[2025-10-16T17:54:10.932+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/chromedriver.py for tasks to queue
[2025-10-16T17:54:10.940+0000] {logging_mixin.py:151} INFO - [2025-10-16T17:54:10.939+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/chromedriver.py
[2025-10-16T17:54:11.070+0000] {logging_mixin.py:151} INFO - [2025-10-16T17:54:11.050+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/chromedriver.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/chromedriver.py", line 6, in <module>
    from py_files.get import ChromebrowserOption, OpenChromeBrowser, StockResults
  File "/opt/airflow/dags/py_files/get.py", line 3, in <module>
    import webdriver
ModuleNotFoundError: No module named 'webdriver'
[2025-10-16T17:54:11.077+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/chromedriver.py
[2025-10-16T17:54:11.266+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/chromedriver.py took 0.354 seconds
[2025-10-16T17:54:41.691+0000] {processor.py:157} INFO - Started process (PID=434) to work on /opt/airflow/dags/chromedriver.py
[2025-10-16T17:54:41.694+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/chromedriver.py for tasks to queue
[2025-10-16T17:54:41.701+0000] {logging_mixin.py:151} INFO - [2025-10-16T17:54:41.700+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/chromedriver.py
[2025-10-16T17:54:41.926+0000] {logging_mixin.py:151} INFO - [2025-10-16T17:54:41.877+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/chromedriver.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/chromedriver.py", line 6, in <module>
    from py_files.get import ChromebrowserOption, OpenChromeBrowser, StockResults
  File "/opt/airflow/dags/py_files/get.py", line 3, in <module>
    import webdriver
ModuleNotFoundError: No module named 'webdriver'
[2025-10-16T17:54:41.940+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/chromedriver.py
[2025-10-16T17:54:42.094+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/chromedriver.py took 0.412 seconds
[2025-10-16T17:55:13.918+0000] {processor.py:157} INFO - Started process (PID=494) to work on /opt/airflow/dags/chromedriver.py
[2025-10-16T17:55:13.921+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/chromedriver.py for tasks to queue
[2025-10-16T17:55:13.930+0000] {logging_mixin.py:151} INFO - [2025-10-16T17:55:13.928+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/chromedriver.py
[2025-10-16T17:55:14.057+0000] {logging_mixin.py:151} INFO - [2025-10-16T17:55:14.034+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/chromedriver.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/chromedriver.py", line 6, in <module>
    from py_files.get import ChromebrowserOption, OpenChromeBrowser, StockResults
  File "/opt/airflow/dags/py_files/get.py", line 3, in <module>
    import webdriver
ModuleNotFoundError: No module named 'webdriver'
[2025-10-16T17:55:14.064+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/chromedriver.py
[2025-10-16T17:55:14.115+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/chromedriver.py took 0.210 seconds
[2025-10-16T17:55:44.833+0000] {processor.py:157} INFO - Started process (PID=553) to work on /opt/airflow/dags/chromedriver.py
[2025-10-16T17:55:44.836+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/chromedriver.py for tasks to queue
[2025-10-16T17:55:44.842+0000] {logging_mixin.py:151} INFO - [2025-10-16T17:55:44.842+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/chromedriver.py
[2025-10-16T17:55:44.938+0000] {logging_mixin.py:151} INFO - [2025-10-16T17:55:44.919+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/chromedriver.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/chromedriver.py", line 6, in <module>
    from py_files.get import ChromebrowserOption, OpenChromeBrowser, StockResults
  File "/opt/airflow/dags/py_files/get.py", line 3, in <module>
    import webdriver
ModuleNotFoundError: No module named 'webdriver'
[2025-10-16T17:55:44.945+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/chromedriver.py
[2025-10-16T17:55:45.005+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/chromedriver.py took 0.180 seconds
[2025-10-16T17:56:15.926+0000] {processor.py:157} INFO - Started process (PID=608) to work on /opt/airflow/dags/chromedriver.py
[2025-10-16T17:56:15.928+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/chromedriver.py for tasks to queue
[2025-10-16T17:56:15.934+0000] {logging_mixin.py:151} INFO - [2025-10-16T17:56:15.933+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/chromedriver.py
[2025-10-16T17:56:16.053+0000] {logging_mixin.py:151} INFO - [2025-10-16T17:56:16.033+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/chromedriver.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/chromedriver.py", line 6, in <module>
    from py_files.get import ChromebrowserOption, OpenChromeBrowser, StockResults
  File "/opt/airflow/dags/py_files/get.py", line 3, in <module>
    import webdriver
ModuleNotFoundError: No module named 'webdriver'
[2025-10-16T17:56:16.059+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/chromedriver.py
[2025-10-16T17:56:16.119+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/chromedriver.py took 0.202 seconds
[2025-10-16T17:56:49.430+0000] {processor.py:157} INFO - Started process (PID=659) to work on /opt/airflow/dags/chromedriver.py
[2025-10-16T17:56:49.433+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/chromedriver.py for tasks to queue
[2025-10-16T17:56:49.440+0000] {logging_mixin.py:151} INFO - [2025-10-16T17:56:49.438+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/chromedriver.py
[2025-10-16T17:56:49.532+0000] {logging_mixin.py:151} INFO - [2025-10-16T17:56:49.516+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/chromedriver.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/chromedriver.py", line 6, in <module>
    from py_files.get import ChromebrowserOption, OpenChromeBrowser, StockResults
  File "/opt/airflow/dags/py_files/get.py", line 3, in <module>
    import webdriver
ModuleNotFoundError: No module named 'webdriver'
[2025-10-16T17:56:49.537+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/chromedriver.py
[2025-10-16T17:56:49.583+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/chromedriver.py took 0.161 seconds
[2025-10-16T17:57:19.834+0000] {processor.py:157} INFO - Started process (PID=717) to work on /opt/airflow/dags/chromedriver.py
[2025-10-16T17:57:19.837+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/chromedriver.py for tasks to queue
[2025-10-16T17:57:19.843+0000] {logging_mixin.py:151} INFO - [2025-10-16T17:57:19.842+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/chromedriver.py
[2025-10-16T17:57:19.953+0000] {logging_mixin.py:151} INFO - [2025-10-16T17:57:19.931+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/chromedriver.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/chromedriver.py", line 6, in <module>
    from py_files.get import ChromebrowserOption, OpenChromeBrowser, StockResults
  File "/opt/airflow/dags/py_files/get.py", line 3, in <module>
    import webdriver
ModuleNotFoundError: No module named 'webdriver'
[2025-10-16T17:57:19.960+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/chromedriver.py
[2025-10-16T17:57:20.013+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/chromedriver.py took 0.186 seconds
[2025-10-16T17:57:50.748+0000] {processor.py:157} INFO - Started process (PID=774) to work on /opt/airflow/dags/chromedriver.py
[2025-10-16T17:57:50.751+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/chromedriver.py for tasks to queue
[2025-10-16T17:57:50.761+0000] {logging_mixin.py:151} INFO - [2025-10-16T17:57:50.760+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/chromedriver.py
[2025-10-16T17:57:50.893+0000] {logging_mixin.py:151} INFO - [2025-10-16T17:57:50.865+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/chromedriver.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/chromedriver.py", line 6, in <module>
    from py_files.get import ChromebrowserOption, OpenChromeBrowser, StockResults
  File "/opt/airflow/dags/py_files/get.py", line 3, in <module>
    import webdriver
ModuleNotFoundError: No module named 'webdriver'
[2025-10-16T17:57:50.902+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/chromedriver.py
[2025-10-16T17:57:50.973+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/chromedriver.py took 0.235 seconds
[2025-10-16T17:58:21.481+0000] {processor.py:157} INFO - Started process (PID=831) to work on /opt/airflow/dags/chromedriver.py
[2025-10-16T17:58:21.485+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/chromedriver.py for tasks to queue
[2025-10-16T17:58:21.493+0000] {logging_mixin.py:151} INFO - [2025-10-16T17:58:21.492+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/chromedriver.py
[2025-10-16T17:58:21.606+0000] {logging_mixin.py:151} INFO - [2025-10-16T17:58:21.586+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/chromedriver.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/chromedriver.py", line 6, in <module>
    from py_files.get import ChromebrowserOption, OpenChromeBrowser, StockResults
  File "/opt/airflow/dags/py_files/get.py", line 3, in <module>
    import webdriver
ModuleNotFoundError: No module named 'webdriver'
[2025-10-16T17:58:21.613+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/chromedriver.py
[2025-10-16T17:58:21.666+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/chromedriver.py took 0.195 seconds
[2025-10-16T17:58:52.698+0000] {processor.py:157} INFO - Started process (PID=890) to work on /opt/airflow/dags/chromedriver.py
[2025-10-16T17:58:52.700+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/chromedriver.py for tasks to queue
[2025-10-16T17:58:52.708+0000] {logging_mixin.py:151} INFO - [2025-10-16T17:58:52.707+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/chromedriver.py
[2025-10-16T17:58:52.820+0000] {logging_mixin.py:151} INFO - [2025-10-16T17:58:52.798+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/chromedriver.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/chromedriver.py", line 6, in <module>
    from py_files.get import ChromebrowserOption, OpenChromeBrowser, StockResults
  File "/opt/airflow/dags/py_files/get.py", line 3, in <module>
    import webdriver
ModuleNotFoundError: No module named 'webdriver'
[2025-10-16T17:58:52.826+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/chromedriver.py
[2025-10-16T17:58:52.877+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/chromedriver.py took 0.189 seconds
[2025-10-16T17:59:23.661+0000] {processor.py:157} INFO - Started process (PID=947) to work on /opt/airflow/dags/chromedriver.py
[2025-10-16T17:59:23.665+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/chromedriver.py for tasks to queue
[2025-10-16T17:59:23.674+0000] {logging_mixin.py:151} INFO - [2025-10-16T17:59:23.673+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/chromedriver.py
[2025-10-16T17:59:23.837+0000] {logging_mixin.py:151} INFO - [2025-10-16T17:59:23.797+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/chromedriver.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/chromedriver.py", line 6, in <module>
    from py_files.get import ChromebrowserOption, OpenChromeBrowser, StockResults
  File "/opt/airflow/dags/py_files/get.py", line 3, in <module>
    import webdriver
ModuleNotFoundError: No module named 'webdriver'
[2025-10-16T17:59:23.849+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/chromedriver.py
[2025-10-16T17:59:23.933+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/chromedriver.py took 0.284 seconds
[2025-10-16T17:59:54.849+0000] {processor.py:157} INFO - Started process (PID=1006) to work on /opt/airflow/dags/chromedriver.py
[2025-10-16T17:59:54.853+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/chromedriver.py for tasks to queue
[2025-10-16T17:59:54.875+0000] {logging_mixin.py:151} INFO - [2025-10-16T17:59:54.874+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/chromedriver.py
[2025-10-16T17:59:55.042+0000] {logging_mixin.py:151} INFO - [2025-10-16T17:59:55.017+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/chromedriver.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/chromedriver.py", line 6, in <module>
    from py_files.get import ChromebrowserOption, OpenChromeBrowser, StockResults
  File "/opt/airflow/dags/py_files/get.py", line 3, in <module>
    import webdriver
ModuleNotFoundError: No module named 'webdriver'
[2025-10-16T17:59:55.049+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/chromedriver.py
[2025-10-16T17:59:55.157+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/chromedriver.py took 0.327 seconds
[2025-10-16T18:00:25.670+0000] {processor.py:157} INFO - Started process (PID=1065) to work on /opt/airflow/dags/chromedriver.py
[2025-10-16T18:00:25.672+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/chromedriver.py for tasks to queue
[2025-10-16T18:00:25.681+0000] {logging_mixin.py:151} INFO - [2025-10-16T18:00:25.680+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/chromedriver.py
[2025-10-16T18:00:25.773+0000] {logging_mixin.py:151} INFO - [2025-10-16T18:00:25.755+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/chromedriver.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/chromedriver.py", line 6, in <module>
    from py_files.get import ChromebrowserOption, OpenChromeBrowser, StockResults
  File "/opt/airflow/dags/py_files/get.py", line 3, in <module>
    import webdriver
ModuleNotFoundError: No module named 'webdriver'
[2025-10-16T18:00:25.780+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/chromedriver.py
[2025-10-16T18:00:25.831+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/chromedriver.py took 0.172 seconds
[2025-10-16T18:00:56.325+0000] {processor.py:157} INFO - Started process (PID=1124) to work on /opt/airflow/dags/chromedriver.py
[2025-10-16T18:00:56.327+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/chromedriver.py for tasks to queue
[2025-10-16T18:00:56.337+0000] {logging_mixin.py:151} INFO - [2025-10-16T18:00:56.336+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/chromedriver.py
[2025-10-16T18:00:56.451+0000] {logging_mixin.py:151} INFO - [2025-10-16T18:00:56.427+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/chromedriver.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/chromedriver.py", line 6, in <module>
    from py_files.get import ChromebrowserOption, OpenChromeBrowser, StockResults
  File "/opt/airflow/dags/py_files/get.py", line 3, in <module>
    import webdriver
ModuleNotFoundError: No module named 'webdriver'
[2025-10-16T18:00:56.464+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/chromedriver.py
[2025-10-16T18:00:56.582+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/chromedriver.py took 0.266 seconds
[2025-10-16T18:01:27.325+0000] {processor.py:157} INFO - Started process (PID=1183) to work on /opt/airflow/dags/chromedriver.py
[2025-10-16T18:01:27.329+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/chromedriver.py for tasks to queue
[2025-10-16T18:01:27.344+0000] {logging_mixin.py:151} INFO - [2025-10-16T18:01:27.343+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/chromedriver.py
[2025-10-16T18:01:33.308+0000] {logging_mixin.py:151} INFO - [2025-10-16T18:01:33.290+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/chromedriver.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/chromedriver.py", line 6, in <module>
    from py_files.get import ChromebrowserOption, OpenChromeBrowser, StockResults
  File "/opt/airflow/dags/py_files/get.py", line 5, in <module>
    import chromedriver_autoinstaller
ModuleNotFoundError: No module named 'chromedriver_autoinstaller'
[2025-10-16T18:01:33.314+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/chromedriver.py
[2025-10-16T18:01:33.365+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/chromedriver.py took 6.060 seconds
[2025-10-16T18:02:41.042+0000] {processor.py:157} INFO - Started process (PID=1261) to work on /opt/airflow/dags/chromedriver.py
[2025-10-16T18:02:41.045+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/chromedriver.py for tasks to queue
[2025-10-16T18:02:41.055+0000] {logging_mixin.py:151} INFO - [2025-10-16T18:02:41.054+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/chromedriver.py
[2025-10-16T18:02:44.623+0000] {logging_mixin.py:151} INFO - [2025-10-16T18:02:44.606+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/chromedriver.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/chromedriver.py", line 6, in <module>
    from py_files.get import ChromebrowserOption, OpenChromeBrowser, StockResults
  File "/opt/airflow/dags/py_files/get.py", line 5, in <module>
    import chromedriver_autoinstaller
ModuleNotFoundError: No module named 'chromedriver_autoinstaller'
[2025-10-16T18:02:44.629+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/chromedriver.py
[2025-10-16T18:02:44.664+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/chromedriver.py took 3.634 seconds
